web/client/components/map/openlayers/MeasurementSupport.jsx

```diff
@@ -84,6 +98,16 @@ class MeasurementSupport extends React.Component {
         var vector;
         var draw;
         var geometryType;
+        let {customStartEndPoint} = newProps.measurement;
```

Comment:
Why do you need these checks? You can not simply set customStartEndPoint (style)? I should name it startEndPointStyle